draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1951,32 +1951,32 @@ deal reasonably well with top-level navigations.
 
 Consider the scenario in which a user reads their email at MegaCorp Inc's
 webmail provider `https://site.example/`. They might expect that clicking on an
-emailed link to `https://projects.example/secret/project` would show them the secret
-project that they're authorized to see, but if `projects.example` has marked their
-session cookies as `SameSite`, then this cross-site navigation won't send them
-along with the request. `projects.example` will render a 404 error to avoid leaking
-secret information, and the user will be quite confused.
+emailed link to `https://projects.example/secret/project` would show them the
+secret project that they're authorized to see, but if `https://projects.example`
+has marked their session cookies as `SameSite=Strict`, then this cross-site
+navigation won't send them along with the request. `https://projects.example`
+will render a 404 error to avoid leaking secret information, and the user will
+be quite confused.
 
 Developers can avoid this confusion by adopting a session management system that
 relies on not one, but two cookies: one conceptually granting "read" access,
-another granting "write" access. The latter could be marked as `SameSite`, and
-its absence would prompt a reauthentication step before executing any
-non-idempotent action. The former could drop the `SameSite` attribute entirely,
-or choose the "Lax" version of enforcement, in order to allow users access to
-data via top-level navigation.
+another granting "write" access. The latter could be marked as `SameSite=Strict`,
+and its absence would prompt a reauthentication step before executing any
+non-idempotent action. The former could be marked as `SameSite=Lax`, or
+`SameSite=None`, in order to allow users access to data via top-level navigation.
```

Comment:
nit: the clause "in order to allow users access to data via top-level navigation" should go with SameSite=Lax, so maybe move it to be closer. And add some more rationale for why SameSite=None might be used, e.g.

``` suggestion
non-idempotent action. The former could be marked as `SameSite=Lax`, in
order to allow users access to data via top-level navigation, or
`SameSite=None`, to permit access in all contexts (including cross-site
embedded contexts).
```